modules/aws/s3_test.go

```diff
@@ -212,3 +245,47 @@ func testEmptyBucket(t *testing.T, s3Client *s3.S3, region string, s3BucketName
 	}
 	require.Equal(t, 0, len((*bucketObjects).Contents))
 }
+
+func addObjectToBucket(t *testing.T, s3Client *s3.S3, region string, s3BucketName string, key string, body string) {
+	logger.Logf(t, "Adding the key '%s' to bucket '%s' with body '%s'", key, s3BucketName, body)
+
+	config := &s3.PutObjectInput{
+		Bucket: aws.String(s3BucketName),
+		Key:    aws.String(key),
+		Body:   aws.ReadSeekCloser(strings.NewReader(body)),
+	}
+
+	_, err := s3Client.PutObject(config)
+
+	if err != nil {
+		t.Fatalf("Failed to add key '%s' to bucket '%s': %s", key, s3BucketName, err.Error())
+	}
+}
+
+func addTagsToObject(t *testing.T, s3Client *s3.S3, region string, s3BucketName string, key string, tags map[string]string) {
+	logger.Logf(t, "Adding the following tags to object '%s': %+v", key, tags)
+
+	// Convert custom tags in AWS tags
```

Comment:
I think you can remove this comment, as it is adding little value to the context here.